frontend/src/Response/SimpleQuestions/RelationQuestion.tsx

```diff
@@ -84,6 +89,9 @@ export default function RelationQuestion(props: {
                     <TextField
                         {...params}
                         size="small"
+                        sx={{ input: {
+                            color: (variants.filter((v) => (v.name === inputValue)).length === 0) ? 'red' : 'inherit'
```

Comment:
`variants.filter((v) => (v.name === inputValue)).length === 0` is used 3 times, so perhaps it will make sense to put it into a separate variable?